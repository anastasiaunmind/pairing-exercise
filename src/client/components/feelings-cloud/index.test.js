import React from "react";
import { render } from "@testing-library/react";
import FeelingsCloud from ".";

test("renders Feelings cloud", () => {
    const items = [
        { value: "happy", checked: false, label: "Happy" },
        { value: "excited", checked: false, label: "Excited" },
    ];
    const { container } = render(<FeelingsCloud items={items} onChange={() => {}} />);

    expect(container).toMatchSnapshot();
});

// todo unit tests