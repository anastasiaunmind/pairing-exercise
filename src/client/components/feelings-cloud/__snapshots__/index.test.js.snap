// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Feelings cloud 1`] = `
<div>
  <div
    class="sc-bdnxRM gAuIVV"
  >
    <div
      class="sc-gtsrHT kgogNg"
      data-checked="false"
      data-testid="feeling"
    >
      Happy
    </div>
    <div
      class="sc-gtsrHT kgogNg"
      data-checked="false"
      data-testid="feeling"
    >
      Excited
    </div>
  </div>
</div>
`;
